README.md

```diff
@@ -19,97 +20,161 @@ This WordPress project is intended to provide:
 - Homebrew (1.2.3)
   - MariaDB (10.0 or greater)
   - Wordpress Command Line (1.2.1)
+  - Composer (1.4.2)
 - PHP 7
+- Codeception for Wordpress (Wp-Browser 1.21)
+
+***
 
 ## Installation
 
-  Installation process for the setting up Wordpress on a local environment.
+Installation process for the setting up Wordpress on a local environment.
+
+- **Create the MySQL user and databases using the MariaDB monitor**.
+
+```
+mysql -uroot
+CREATE DATABASE chappress_dev;
+GRANT ALL PRIVILEGES ON chappress_dev.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+
+CREATE DATABASE chappress_test;
+GRANT ALL PRIVILEGES ON chappress_test.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
+
+CREATE DATABASE chappress_test_wpunit;
+GRANT ALL PRIVILEGES ON chappress_test_wpunit.* TO "chappress"@"localhost" IDENTIFIED BY "chappress";
```

Comment:
I encountered many problems after I added in the test port on fri. Tests were working but the site wouldn't load, or after editing, the site would load but the tests wouldn't. So l tried the latest setup to work with these issues. 

I'll look at the link and see I can make it work like that. 

So the setup I have, is to test on a demo site and the original public isn't compromised.  But from the link you sent, looks like it can be configured all from the same wordpress install.

 So we are testing the `/public` which will also be the live site?

Note for the wploader:
It's just an optional suite that was installed during the setup wizard. It can be removed if need be or added again later. It seems to combine  Codeception's unit testing with the Wordpress core testing suite.  

As noted in the docs: 

> The generated test case extends the WPTestCase class and it exposes all the methods defined by Codeception\Test\Unit test case and the Core suite \WP_UnitTestCase. 

> It has access WordPress defined functions and classes in integration tests; a wrapping of the WordPress PhpUnit based test suite provided in the WordPress repository